src/main/java/utils/ExportResultsUtil.java

```diff
@@ -0,0 +1,74 @@
+package utils;
+
+import refactoring.MoveMethodRefactoring;
+
+import com.intellij.psi.PsiMethod;
+import com.intellij.psi.PsiClass;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.function.Function;
+import java.util.Optional;
+
+import static java.nio.file.Files.createFile;
+import static java.nio.file.Files.write;
+import static utils.PsiUtils.getHumanReadableName;
+
+/**
+ * Class exports (MoveMethod) refactoring results to the file
+ */
+public class ExportResultsUtil {
+
+    //private static final Logger LOGGER = Logging.getLogger(ExportResultsUtil.class);
+
+
+    /**
+     * Method exports results of refactoring to the file (Now support only MoveMethodRefactoring)
```

Comment:
You can generalize the method by using generics. 
`List<? extends Refactoring> refactorings` 
There is an interface for all types of refactorings, merge master into this branch to see it.